python/ray/autoscaler/kuberay/ray-cluster.complete.yaml

```diff
@@ -36,6 +36,8 @@ spec:
       # Use `resources` to optionally specify custom resource annotations for the Ray node.
       # The value of `resources` is a string-integer mapping.
       # Currently, `resources` must be provided in the unfortunate format demonstrated below.
+      # Moreover, "CPU" and "GPU" should NOT be included in the `resources` arg.
+      # (Use `num-cpus` and `num-gpus` rayStartParams instead.)
```

Comment:
This fact is completely unclear and has led to some user confusion.

Ray resource specification will need some clean-up in the near future.